src/raven.js

```diff
@@ -777,11 +777,16 @@ Raven.prototype = {
             var xhrproto = XMLHttpRequest.prototype;
             fill(xhrproto, 'open', function(origOpen) {
                 return function (method, url) { // preserve arity
-                    this.__raven_xhr = {
-                        method: method,
-                        url: url,
-                        status_code: null
-                    };
+
+                    // intentionally checking against 0 here (start of string)
+                    if (url.indexOf(self._globalEndpoint) !== 0) {
```

Comment:
I could also use `parseUrl` and verify that host name and path match, omitting protocol.